roles/openshift_prometheus/tasks/install_prometheus.yaml

```diff
@@ -0,0 +1,223 @@
+---
+
+# namespace
+- name: Add prometheus project
+  oc_project:
+    state: "{{ state }}"
+    name: "{{ openshift_prometheus_namespace }}"
+    node_selector: "{{ openshift_prometheus_node_selector | replace(':','=') | replace('{', '') | replace('}', '') | replace(' ', '') | replace(\"u\'\", '') | replace(\"\'\", '') }}"
```

Comment:
dict_to_string is not enough because you also need to strip quotes and braces. it's a very proprietary formatting so I don't this it justifies its own plugin. I think adding support in oc_project is a better way to go.